_sass/module/_landing-description.scss

```diff
@@ -3,15 +3,19 @@
   color: $color-grey-2;
   font-style: italic;
   font-weight: 400;
+  text-align: center;
+  margin: auto;
+  margin-bottom: 5px;
 }
 
 .landing__description {
   padding: 50px;
 }
 
 .landing__description__paragraph {
-  display: flex;
-  justify-content: center;
```

Comment:
I think it's right to delete this, but I notice that the landing page scrolls horizontally on smaller screens. This is at 700px for example:
![screenshot from 2018-09-08 11-12-54](https://user-images.githubusercontent.com/509703/45257272-2ab81080-b358-11e8-838e-d50b6ac5f240.png)

Reply:
Okay, flex display and justified content rules have been deleted here on my branch. I have pushed my most recent work. The view at 700px looks ok to me, but let me know if it's still scrolling horizontally for you.